skyfield/tests/test_stars.py

```diff
@@ -1,8 +1,25 @@
+from pyongc import data
 from skyfield import api
-from skyfield.data.hipparcos import load_dataframe
+from skyfield.data import hipparcos, ongc
 
 def test_dataframe():
     with api.load.open('hip_main.dat.gz') as f:
-        df = load_dataframe(f)
+        df = hipparcos.load_dataframe(f)
     star = api.Star.from_dataframe(df)
     assert repr(star) == 'Star(ra shape=9933, dec shape=9933, ra_mas_per_year shape=9933, dec_mas_per_year shape=9933, parallax_mas shape=9933, epoch shape=9933)'
+
+def test_dataframe_from_ongc():
+    ongc_data = data.all()
+    df = ongc.load_dataframe(ongc_data)
+    hercules = api.Star.from_dataframe(df.loc['NGC6205'])
+    assert repr(hercules) == (
+        'Star(ra=250.42345833333337, dec=36.46130555555556, ra_mas_per_year=-3.18, '
```

Comment:
Normally I would object to a `repr()` since even very small changes in how numbers are input and rounded will break the tests, but in this case it's a bit elegant—it lets us very quickly verify that all the essential properties of the star are correct. So let's keep it this way, and see if it ever breaks!